velox/functions/prestosql/types/IPPrefixType.cpp

```diff
@@ -120,6 +148,33 @@ class IPPrefixCastOperator : public exec::CastOperator {
     });
   }
 
+  static void castFromIpAddress(
+      const BaseVector& input,
+      exec::EvalCtx& context,
+      const SelectivityVector& rows,
+      BaseVector& result) {
+    auto* rowVectorResult = result.as<RowVector>();
+    const auto* ipAddrVector = input.as<SimpleVector<int128_t>>();
+
+    context.applyToSelectedNoThrow(rows, [&](auto row) {
+      auto intIpAddr = ipAddrVector->valueAt(row);
+      const auto tryPrefixLength =
```

Comment:
I don't think this is particularly necessary as the entire range of the 16 byte array are all valid ips.

Reply:
Hm make sense, but I still need to get the prefix length. I don't think the cost of the check is expensive